### ISSUE TITLE
Email templates chooser: sort by name!

### ISSUE THREAD

**Issue by EZS-JD** [salesagility/SuiteCRM#2075]
#### Issue

Whenever I can choose an Email template (e.g. workflow, campaign) the dropdown with the email templates is not sorted in any way I can figure out. With already hundreds of email templates in the system it gets very messy and hard to find what you're looking for.
#### Expected Behavior

Either Sort by name (simple)
Or change selection interface from dropdown to an autocompletion chooser (as when choosing a linked contact e.g.)
#### Your Environment

- SuiteCRM Version used: 7.4.3
- Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)):
- Environment name and version (e.g. MySQL, PHP 7):
- Operating System and version (e.g Ubuntu 16.04):

**Comment by virtualme123** [comment]
I too think this would be a good feature as the email list is quite messy and the order is a bit it and miss.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Email templates chooser: sort by name!](https://trello.com/c/aixMNQxN/80-email-templates-chooser-sort-by-name)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
